server/common.js

```diff
@@ -187,6 +188,44 @@ var getDate = function () {
 }
 exports.getDate = getDate;
 
+/**
+* return the current date
+*
+* @return {date}
+*/
+var getDateObject = function(){
+    return new Date();
+}
+exports.getDateObject = getDateObject;
+
+/**
+* return the time in "<0> days, <0> hours, <0> minutes, <0> seconds" format
+*
+* @return {string}
+*/
+var getTime = function(time){
+    const date = new Date(time);
+    var timeString = '';
+    const days = date.getUTCDate()-1;
```

Comment:
Why do we need to be checking `if(variable)`

Reply:
So that if the value is 0 it doesn't get added to the `timeString`

Say time left is 25 mins 3 seconds
It will return 25 mins, 3 seconds. Instead of 0 Days, 0 Hours, 25 mins, 3 seconds.